tree/dataframe/inc/ROOT/RDF/ColumnReaderUtils.hxx

```diff
@@ -84,7 +84,7 @@ MakeColumnReader(unsigned int slot, RDefineBase *define,
 /// incorrectly from a compiled MakeColumnReaders symbols when invoked from a jitted symbol.
 struct RColumnReadersInfo {
    const std::vector<std::string> &fColNames;
-   const RColumnRegister &fCustomCols;
```

Comment:
Potential "issue" that I realize now with this change: couldn't that break user's existing code?

Reply:
no, this is all internal